README.md

```diff
@@ -0,0 +1,37 @@
+# ElasticSearch on Rancher
+
+> Rancher configuration for running an Elasticsearch cluster
+
+## Services deployed:
+
+- `es-master`: Elasticsearch nodes, part of the Elasticsearch cluster, one runs on every host that has an `escluster=true` tag defined.
+- Justwatch [Exporter]: Exposes Prometheus metrics on the cluster status.
+- es-client: Rancher cattle DNS service, which points the `es-client` domain name to `es-master`, for backwards compatibility with existing services.
+
+## Deployment
+
+Deployment is driven by Travis (Dev and PR review deployments) and TeamCity (staging and production deployments). We use the NHS.UK [CI-Deployment] repo for deployment to the Rancher environment, this sets the following ENV variables.
```

Comment:
Maybe a link? (e.g. `[Travis](https://travis-ci.org/nhsuk/rancher-elasticsearch-cluster)`) and for TeamCity too.

Reply:
Happy to provide the Travis link.